src/DynamoCore/Models/RunSettings.cs

```diff
@@ -73,6 +75,36 @@ public bool RunEnabled
             }
         }
 
+        /// <summary>
+        /// This property will enable or disable the ComboBox RunTypes
+        /// </summary>
+        public bool RunTypesEnabled
```

Comment:
Does this property control the file trust dialog being shown or hidden?

Reply:
Same, which combo box is being controlled by this?

Reply:
yes, it controls only if the ComboBox RunTypes is enabled or disabled
![image](https://user-images.githubusercontent.com/61755417/170340553-071aec52-1742-4619-9c36-e66189c615ea.png)